src/TestInfrastructure/TestInfrastructure.csproj

```diff
@@ -328,11 +328,95 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="**\obj\**\*" />
+    <Compile Include="Framework\ConfigurableActiveSolutionBoundTracker.cs" />
```

Comment:
What's our normal practice on using wildcards in project files?
VS2015 doesn't handle them very well and tends to expand the list; VS2017 should handle wildcards correctly.